### PR TITLE
T18 - Armstrong number.

### PR DESCRIPTION
**Pull request by Priyanka15-12**

### CODE REVIEW

**Comment by MetaNaveen** [comment]
Can you assign it your current Mentor as both 18 and 18.1 are kind of similar, and I have not yet attended the PR.?

File: Program.cs [hunk 1]
```diff
@@ -1,7 +1,18 @@
...
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
+   internal class Program {
+      static void Main (string[] args) {
+         Console.Write ("Enter a positive number: ");
+         int n = int.Parse (Console.ReadLine ());
+         int temp = n;
+         Console.Write ($"{n} is a {(armstrong (n) == temp ? "" : "in")}valid armstrong number.");
```
Comment (meenashriTMM) [inline Program.cs:7]: 'I' should be capital in 'Invalid'

File: Program.cs [hunk 2]
```diff
@@ -1,7 +1,18 @@
...
+      }
+      static int armstrong (int n) {
```
Comment (meenashriTMM) [inline Program.cs:9]: Method name should always start with capital letter

Comment (meenashriTMM) [inline Program.cs:9]: Leave an empty line before method

File: Program.cs [hunk 3]
```diff
@@ -1,7 +1,18 @@
...
+         int sum = 0, intlen = n.ToString ().Length;
```
Comment (meenashriTMM) [inline Program.cs:10]: Please follow CamelCase naming convention. i.e variable name should be intLen

**Review by meenashriTMM — CHANGES_REQUESTED** [review]
Formatting is incorrect

**Comment by meenashriTMM** [comment]
@Priyanka15-12  Fix the review changes and then assign me.

**Review by meenashriTMM — APPROVED** [review]

**Pull request closed without merging**